src/i18n.ts

```diff
@@ -0,0 +1,56 @@
+import i18n from "i18next";
+import { initReactI18next } from "react-i18next";
+
+const resources: Record<string, { translation: Record<string, string> }> = {
+  en: {
+    translation: {
+      language: 'EN',
```

Comment:
I'm not sure if I would use this for implementing logic, like the language codes. So I'd rather remove that and place the EN / DE switch elsewhere in some kind of languageCodes map.,